src/Input/Input.stories.svelte

```diff
@@ -0,0 +1,315 @@
+          type: 'select'
+        },
+        options: [
+          'checkbox',
+          'color',
+          'date',
+          'datetime-local',
+          'email',
+          'file',
+          'number',
+          'password',
+          'radio',
+          'range',
+          'search',
+          'select',
+          'switch',
+          'text',
+          'textarea',
+          'time',
+          'url'
+        ]
+      },
+      plaintext: {
+        control: {
+          type: 'boolean'
+        },
+        table: {
+          disable: false
+        }
+      },
+      size: {
+        control: {
+          type: 'select'
+        },
+        options: ['sm', 'lg'],
+        table: {
+          disable: true
+        }
+      },
+      color: {
+        control: 'color',
+        table: {
+          disable: true
+        }
+      },
+      feedback: {
+        control: 'text',
+        table: {
+          disable: true
+        }
+      },
+      disabled: {
+        control: 'boolean'
+      },
+      checked: {
+        control: 'boolean',
+        table: {
+          disable: true
+        }
+      },
+      files: {
+        control: 'array',
+        table: {
+          disable: true
+        }
+      },
+      group: {
+        control: 'text',
+        table: {
+          disable: true
+        }
+      },
+      inner: {
+        control: 'text',
+        table: {
+          disable: true
+        }
+      },
+      label: {
+        control: 'text',
+        table: {
+          disable: true
+        }
+      },
+      multiple: {
+        control: 'boolean',
+        table: {
+          disable: true
+        }
+      },
+      name: {
+        control: 'text',
+        table: {
+          disable: true
+        }
+      },
+      placeholder: {
+        control: 'text'
+      },
+      readonly: {
+        control: 'boolean',
+        table: {
+          disable: true
+        }
+      },
+      reverse: {
+        control: 'boolean'
+      },
+      value: {
+        control: 'text'
+      }
+    },
+    args: {
+      type: 'text',
+      plaintext: false,
+      invalid: false,
+      valid: false,
+      bsSize: undefined,
+      disabled: false,
+      placeholder: 'placeholder',
+      reverse: false,
+      value: ''
+    }
+  };
+</script>
+
+<script>
+  import { Story, Template } from '@storybook/addon-svelte-csf';
+  import { FormGroup, FormText, Label } from '@sveltestrap/sveltestrap';
+
+  let changeValue = '';
+  let focused = false;
+  let inner = '';
+  let inputValue = '';
+  let radioGroup;
+
+  const resize = () => {
+    inner.style.height = 'auto';
+    inner.style.height = 4 + inner.scrollHeight + 'px';
+  };
+
+  const changeEvent = (e) => {
+    changeValue = e.target.value;
+  };
+
+  const inputEvent = (e) => {
+    inputValue = e.target.value;
+  };
+</script>
+
+<Template let:args>
+  <Input {...args} />
+</Template>
+
+<Story name="Basic" />
+
+<Story name="Text">
+  <div class="input-example">
```

Comment:
I would restrict the width of all the examples to something like 300px or 400px. it looks ugly spanning the entire length

Reply:
this include inputgroup and other stories